client/evilwallet/wallets.go

```diff
@@ -14,11 +14,14 @@ import (
 
 type walletID int
 type WalletType int8
+type WalletStatus int8
 
 const (
-	// fresh is used for automatic Faucet Requests, outputs are returned one by one
 	other WalletType = iota
+	// fresh is used for automatic Faucet Requests, outputs are returned one by one
 	fresh
+	// reuse stores resulting outputs of double spends or transactions issued by the evilWallet
+	reuse
```

Comment:
When is a wallet labeled as `reuse`?

Reply:
The reuse wallet should contain outputs that are the leafs in the custom conflict tree - outputs that were not used during the custom conflict creation. The idea is to store them in a separate wallet for further reuse in spamming - we may provide custom conflicts struct as a spam secenario and instruct that spammer can reuse those outputs for deep spamming.
However I put this just to remember the idea, it's not implemented, we might discuss if it's the right direction, otherwise we can remove this category.